pkg/webui/components/radio-button/radio.js

```diff
@@ -107,20 +108,22 @@ class RadioButton extends React.PureComponent {
       radioProps.name = this.context.name
       radioProps.disabled = disabled || this.context.disabled
       radioProps.checked = value === this.context.value
+      radioProps.id = `${this.context.name}-${stringToHash(value)}`
```

Comment:
Why `stringToHash` ?

Reply:
In order to make the ID unique, I use a combination of the name and value. However, the value could be an arbitrary string like `Example Value`, which would break the ID, so I use the hash instead.

Reply:
There a couple things to that:

1. I cannot imagine a use case in the console/oauth apps where one might need to define `Example Value` as the value. We dont use native form submission and do not allow the users to use the console/oauth without js, hence we can always transform the value and adjust the result accordingly.
2. In general it doesnt look right to use `Example Value` as the value for a radio button (imho)
3. `{name}-123123123123` is just noise in the dom and I personally check dom elements when adding features (imho)
4. `stringToHash` is just some function most probably copied from the internet without any references and I dont really feel to examine it carefully to argue about its collision probability
5. There is no reason to add the id attribute in the first place as `<input />` is wrapped within the `<label />` tag which implicitly associates those two elements.
6. You can always check and remove whitespaces via small utility function within the component, there no reason to add a separate dependency for a radio button. The less dependencies our core components have, the better.